occurrences/management/commands/send_enrolment_summary.py

```diff
@@ -1,8 +1,7 @@
+from django.conf import settings
 from django.core.management.base import BaseCommand
 from occurrences.models import Enrolment
 
-from palvelutarjotin import settings
```

Comment:
I quickly checked that same import is done in a few other places as well. Maybe those should also be fixed/changed like this?